opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/EpsilonMapping.java

```diff
@@ -0,0 +1,48 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+import java.util.Map;
+import java.util.Set;
+
+import org.opt4j.core.Individual;
+import org.opt4j.core.Objective;
+import org.opt4j.core.Objectives;
+
+import com.google.inject.ImplementedBy;
+
+/**
+ * The {@link EpsilonMapping} implements the ε mapping used by the
+ * {@link AeSeHSelector}.
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+@ImplementedBy(AdditiveEpsilonMapping.class)
+public interface EpsilonMapping {
```

Comment:
Is it likely that we get another EpsilonMapping? To me, this also looks extremely AeSeH specific. If not, we could remove the interface in favor for just a single class.